### DISCUSSION TITLE
There is no search feature on the zen mods web page

### DISCUSSION THREAD

**Discussion by universish** [zen-browser/desktop#3622] [General]
The zen mods web page does not have the ability to search for words in the zen mod name, zen mod tag, zen mod description or title. 
We can do a word search with Ctrl+F. But at least we should be able to search for zen mod name and zen mod tag. 
When this project grows and hundreds of zen mods are added to the project, we will need this search button.
website url: https://zen-browser.app/mods/

**Comment by kgarrett67** [comment]
I was thinking it was there and now it's gone. Although `ctrl+F` works, I did like the search box at the top of the page.

**Reply by rubaboo** [reply]
Sorting by date was there and is now gone. Used to be you could sort the mods by published date as well as updated date. 😪
